packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5507,8 +5501,8 @@ export default class Meeting extends StatelessWebexPlugin {
         })
       )
       .then(() => {
-        if (this.mediaProperties.hasLocalShareTrack()) {
-          return this.enqueueScreenShareFloorRequest();
+        if (localStreams?.screenShare?.video) {
```

Comment:
this looks wrong, is this a merge gone wrong? we should be using `hasLocalShareStream()` and `enqueueScreenShareFloorRequest()` here

Reply:
Looks like a merge gone wrong. Sorry about that. Done in #3053